pkg/localdata/config.go

```diff
@@ -0,0 +1,50 @@
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package localdata
+
+import (
+	"os"
+	"path"
+
+	"github.com/BurntSushi/toml"
+)
+
+type configBase struct {
+	file string
+}
+
+// TiUPConfig represent the config file of TiUP
+type TiUPConfig struct {
+	configBase
+	Mirror string `toml:"mirror"`
+}
+
+// InitConfig returns a TiUPConfig struct which can flush config back to disk
+func InitConfig(root string) *TiUPConfig {
+	config := TiUPConfig{configBase{path.Join(root, "tiup.toml")}, ""}
```

Comment:
I think it's better to reset the mirror when the `tiup.toml` doesn't exist, which means it's the first time to call `Init'Config`.